ensime_shared/ensime.py

```diff
@@ -48,13 +50,19 @@ def __init__(self, vim):
         # race condition of autocommand handlers being invoked as they're being
         # defined.
         self._vim = vim
+        self._ticker = None
         self.clients = {}
 
     @property
     def using_server_v2(self):
         """bool: Whether user has configured the plugin to use ENSIME v2 protocol."""
         return bool(self.get_setting('server_v2', 0))
 
+    def is_buffer_ensime_compatible(self):
```

Comment:
Please move this method to `Editor`, we are trying to reduce the dependency on the Vim object to only the editor module. This is the best for many reasons, but mostly for making the testing easier.